app/Http/Controllers/OrderInvoice/EditController.php

```diff
@@ -108,24 +106,56 @@ public function update(Request $request)
                     $pnss_updated_at = now();
                     $payment_notice_sending = isset($data['payment_notice_sending']) ? 1 : 0;
 
-                    echo '9';
-
-                    for ($i = 0; $i < count($data['item']); $i ++) {
-                        echo '10';
-                        DB::table('invoices_table')->where('id', $invoice_id)->update([
-                            'item' => $data['item'][$i]
+                    //追加箇所
+                    //insertで追加されず、上書きされる
+                    foreach($data['item'] as $item){
+                        DB::table('items_table')->updateorInsert(
```

Comment:
```suggestion
                        DB::table('items_table')->updateOrCreate(
```
こっち使ってみて試してもらえませんか…？